rest-service/manager_rest/test/endpoints/test_executions.py

```diff
@@ -767,3 +770,128 @@ def test_install_empty_blueprint(self):
         self.assertEquals(get_execution['blueprint_id'], blueprint_id)
         self.assertEquals(get_execution['deployment_id'],
                           deployment_response['id'])
+
+    def _execution_resume_test(self, deployment, status):
+        execution = self.sm.put(models.Execution(
+            id=str(uuid.uuid4()),
+            _deployment_fk=deployment._storage_id,
+            created_at=datetime.now(),
+            is_system_workflow=False,
+            workflow_id='install',
+            status=status
+        ))
+        tasks_graph = self.sm.put(models.TasksGraph(
+            _execution_fk=execution._storage_id,
+            name='install',
+            created_at=datetime.now()
+        ))
+        operation = self.sm.put(models.Operation(
+            _tasks_graph_fk=tasks_graph._storage_id,
+            parameters={'current_retries': 20},
+            state=cloudify_tasks.TASK_FAILED,
+            created_at=datetime.now()
+        ))
+
+        self.client.executions.resume(execution.id, force=True)
+
+        operations = self.sm.list(models.Operation, get_all_results=True)
+        self.assertEqual(len(operations), 1)
+        operation = operations[0]
+        self.assertEqual(operation.state, cloudify_tasks.TASK_PENDING)
+        self.assertEqual(operation.parameters['current_retries'], 0)
+
+        execution = self.sm.get(models.Execution, execution.id)
+        # started or terminated, it might have already finished by the time
+        # the test was done
+        self.assertIn(execution.status,
+                      (ExecutionState.STARTED, ExecutionState.TERMINATED))
+
+    def test_resume_force_failed(self):
+        """Force-resume resets operation state and restart count"""
```

Comment:
there is repetitiveness in the tests, but I don't like to write more framework for few tests